aea/cli/common.py

```diff
@@ -379,7 +379,7 @@ def try_get_item_target_path(
 
     :return: the item target path
     """
-    target_path = os.path.join(path, author_name, item_type_plural, item_name)
+    target_path = os.path.join(path, "vendor", author_name, item_type_plural, item_name)
```

Comment:
The function is called `try_get_item_target_path`. I am worried that this is ambiguous. It should probably be renamed to `try_get_vendorized_item_target_path` to make clear that this is only for items in the vendor folder.

Reply:
But we came to a conclusion that target (the thing that we add/fetch) is always a published thing so it should go to `vendor` folder. Maybe that assumption is enough and we don't need that long naming in this case?)

Reply:
Well but another developer might easily confuse it and use it wrongly, I think in this case it would help to replace the names